src/components/grid/PCol.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div :class="className">
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'PCol',
+  props: {
+    sizeXs: Number,
+    sizeSm: Number,
+    sizeMd: Number,
+    sizeLg: Number,
+    sizeXl: Number
+  },
+  computed: {
+    className() {
```

Comment:
Update to be like https://github.com/vuejs-norte/puto-framework/blob/master/src/components/PButton.vue#L40-L46